roxctl/netpol/generate/generate.go

```diff
@@ -0,0 +1,23 @@
+package generate
+
+import (
+	"github.com/np-guard/cluster-topology-analyzer/pkg/controller"
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/pkg/protoconv/networkpolicy"
+)
+
+func (cmd *netpolGenerateCommand) generateNetpol() error {
+	recommendedNetpols, err := controller.PoliciesFromFolderPath(cmd.folderPath)
+	if err != nil {
+		return errors.Errorf("Error synthesizing policies from folder: %v", err)
+	}
+
+	for _, netpol := range recommendedNetpols {
+		yamlPolicy, err := networkpolicy.KubernetesNetworkPolicyWrap{NetworkPolicy: netpol}.ToYaml()
+		if err != nil {
```

Comment:
Question: could we `log` the error and skip adding it to the output? Or the plan is to abort the process entirely if a single NetworkPolicy is incorrect?

Reply:
Right now, I lean towards breaking the whole process to not mask that we might be missing Network Policies.
If this is run by CI, no one might ever see the warning.

Reply:
Got it